turbo.json

```diff
@@ -1,31 +1,44 @@
 {
   "$schema": "https://turbo.build/schema.json",
-  "globalDotEnv": [".env.local"],
-  "pipeline": {
+  "remoteCache": {
+    "signature": true
+  },
+  "globalDependencies": [
+    ".env.local"
+  ],
+  "tasks": {
     "topo": {
-      "dependsOn": ["^topo"]
+      "dependsOn": [
+        "^topo"
+      ]
     },
     "build": {
-      "dependsOn": ["^build"],
+      "env": ["AUTH_SECRET", "BIGCOMMERCE_*"],
```

Comment:
@bigcommerce/team-catalyst FYI, turbo doesn't inject environment variables by default anymore, so you need to define them here. There are some ones that it automatically injects, e.g. [framework env vars](https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#framework-inference)